### PR TITLE
Add Bugsnag JavaScript integration

### LINKED ISSUE #689: Add JS-side Bugsnag

### PR DESCRIPTION
**Pull request by kberzinch**
Closes #689

### CODE REVIEW

File: resources/js/app.js [hunk 1]
```diff
@@ -4,14 +4,24 @@
...
+})
+
 require('./bootstrap');
 var axios = require('axios');
 
 //Import SweetAlert2 for nice alert dialogs
 import Swal from 'sweetalert2';
 window.Swal = Swal;
 
-window.Vue = require('vue');
```
Comment (JoshOldenburg) [inline resources/js/app.js:14]: Is this line replaced by the following lines?

Comment (kberzinch) [reply resources/js/app.js:14]: Line 7 AFAIK but I'm not great at JS

**Review by JoshOldenburg — APPROVED** [review]
I missed that line.

File: resources/js/app.js (hunk 1 above)
Comment (JoshOldenburg) [reply resources/js/app.js:14]: I missed that line.

**Pull request merged**